ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard.component.ts

```diff
@@ -59,13 +59,18 @@ export class DashboardComponent implements OnInit, OnDestroy {
     private anchor: Subscription = new Subscription();
 
     constructor(
-        private router: Router,
+        protected router: Router,
```

Comment:
Just wondering why you need `protected` here (and in line 65) ?

Reply:
because we start using router and session in the DashboardRedesignedComponent. I think it's better than service instance renaming